graphql-tool-idl-plugin/src/test/resources/graphql-client-projects/project_10/main.bal

```diff
@@ -0,0 +1,4 @@
+public function main() {
+    client "./graphql.config.yaml" as foo;
```

Comment:
Is this intended to be a positive test case? if so, local client declarations won't work IINM